File: BplMubarak.swift
import Foundation

if let T = Int(readLine()!){
    for _ in 0..<T{
        if let S = readLine(){
            var legalBalls = 0

            for char in S{
                if char.isNumber || char == "O"{
                    legalBalls += 1
                }
            }

            let overs = legalBalls/6
            let balls = legalBalls%6

            var result = ""
            if overs > 0{
                result += "\(overs) OVER"
                if overs > 1 {result += "S"}
            }
            if balls>0{
                if !result.isEmpty{result += " "}
                result += "\(balls) BALL"
                if balls>1{result += "S"}
            }
            print(result)
        }
    }
}

File: BplMubarak.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{

int T;
cin>>T;


    while(T--){
        string S;
        cin>>S;

        int legalBalls = 0;

        for(char ch : S){
            if((ch>='0' && ch<='6') || ch=='O'){
                legalBalls++;
            }
        }

    int overs = legalBalls/6;
    int balls = legalBalls%6;

    if(overs>0){
        cout<<overs<<" OVER";
        if(overs>1) cout<<"S";
    }
    if(balls>0){
        if(overs>0) cout<<" ";
        cout<<balls<<" BALL";
        if(balls>1) cout<<"S";
    }
    cout<<endl;
    }

    return 0;
}